src/instructlab/model/evaluate.py

```diff
@@ -335,6 +331,7 @@ def launch_server(
     "--model",
     type=click.STRING,
     cls=clickext.ConfigOption,
+    required=True,
```

Comment:
I thought the reason this wasn't required here was because it's in the configuration as well.

Reply:
required=True is the workflow we use for flags populated by the cfg bc they should and will always be there, right?